tests/test_failures.py

```diff
@@ -200,10 +200,6 @@ def dead_letter_workflow() -> None:
     assert exc_info.errisinstance(DBOSDeadLetterQueueError)
     assert handle.get_status().status == WorkflowStatusString.RETRIES_EXCEEDED.value
 
-    with SetWorkflowID(handle.get_workflow_id()):
-        DBOS.start_workflow(dead_letter_workflow)
-    assert recovery_count == max_recovery_attempts + 2
```

Comment:
This path does not make sense anymore (you can't call a workflow in "non recovery mode")